source/renderer/app/i18n/locales/ja-JP.json

```diff
@@ -571,7 +571,7 @@
   "wallet.transaction.assuranceLevel.high": "高",
   "wallet.transaction.assuranceLevel.low": "低",
   "wallet.transaction.assuranceLevel.medium": "中",
-  "wallet.transaction.confirmations": "{confirmationsNumber, plural, one {# 確認} =21 {#+ 確認} other {# 確認}}",
+  "wallet.transaction.confirmations": "{confirmationsNumber, plural, one {# 確認} =21 {20+ 確認} other {# 確認}}",
```

Comment:
@DeeJayElly are we sure the Japanese plural is the same as singular?

Reply:
Yes, its confirmed.

<img width="1116" alt="Screenshot 2019-08-13 03 51 54" src="https://user-images.githubusercontent.com/15862062/62928207-b5ca3380-bd7d-11e9-82ad-499d8cee339a.png">

Reply:
Perfect! Thanks!